site/docs/reference/Connectors/capture-connectors/google-analytics-4.md

```diff
@@ -0,0 +1,184 @@
+# Google Analytics 4
+
+This connector captures data from Google Analytics 4 properties into Flow collections via the
+[Google Analytics Data API](https://developers.google.com/analytics/devguides/reporting/data/v1).
+
+:::info
+This connector supports Google Analytics 4, not Universal Analytics.
+
+Universal Analytics is supported by a [separate connector](./google-analytics.md).
+:::
+
+It’s available for use in the Flow web application. For local development or open-source workflows, [`ghcr.io/estuary/source-google-analytics-data-api:dev`](https://ghcr.io/estuary/source-google-analytics-data-api:dev) provides the latest version of the connector as a Docker image. You can also follow the link in your browser to see past image versions.
+
+This connector is based on an open-source connector from a third party, with modifications for performance in the Flow system.
+You can find their documentation [here](https://docs.airbyte.com/integrations/sources/google-analytics-v4/),
+but keep in mind that the two versions may be significantly different.
+
+## Supported data resources
+
+The following data resources are supported:
+
+* Daily active users
+* Devices
+* Four-weekly active users
+* Locations
+* Pages
+* Traffic sources
+* Website overview
+* Weekly active users
+
+Each is [fetched as a report](https://developers.google.com/analytics/devguides/reporting/data/v1/rest/v1beta/properties/runReport) and mapped to a Flow collection through a separate binding.
+
+You can also capture [custom reports](#custom-reports).
+
+## Prerequisites
+
+To use this connector, you'll need:
+
+* The Google Analytics Reporting API [enabled](https://support.google.com/googleapi/answer/6158841?hl=en) on your Google [project](https://cloud.google.com/storage/docs/projects) with which your Analytics property is associated.
```

Comment:
You need to enable Google Analytics Data API (as well? not sure if Reporting API is necessary)

Reply:
Whoops, thanks for catching that